lib/gs-plugin-loader.c

```diff
@@ -1250,8 +1250,9 @@ gs_plugin_loader_app_is_valid (GsApp *app, gpointer user_data)
 		return FALSE;
 	}
 
-	/* don't show blacklisted apps */
-	if (gs_app_has_category (app, "Blacklisted")) {
+	/* don't show blacklisted apps if they are not from USB*/
+	if (gs_app_has_category (app, "Blacklisted") &&
+		!gs_app_has_category (app, "USB")) {
```

Comment:
I don't think we should add changes to the plugin loader for this. Even though this is a simpler change, every change we add to the plugin loader will make it more complex and make rebases more difficult.

Reply:
Thinking better about it. I could be okay with this change if adding it to the EOS plugin is much more complex. i.e. Have you verified if overriding `gs_plugin_add_category_apps` and modifying the `gs_plugin_refine` makes it too complex?

Reply:
Will try that!